testing/benchmark/variables.tf

```diff
@@ -68,6 +68,19 @@ variable "docker_image_override" {
   type = map(string)
 }
 
+variable "apm_shards" {
+  default = {
+    "traces-apm@custom" : 1,
+    "logs-apm.app@custom" : 1,
+    "logs-apm.error@custom" : 1,
+    "metrics-apm.app@custom" : 1,
+    "metrics-apm.internal@custom" : 1,
+    "traces-apm.rum@custom" : 1,
+    "traces-apm@custom" : 1,
+  }
+  type = map(string)
+}
```

Comment:
Could we not set anything at all by default? I would prefer if the default came from the integration package, so we don't inadvertently test with something other than the actual defaults. i.e. if we were to start setting number_of_shards in the package, then our benchmarks should pick that up.

Reply:
yep, that sounds better. I'll remove the default.